.github/workflows/data_update_v2.yml

```diff
@@ -105,3 +119,9 @@ jobs:
           Created with commit ${{env.SCRAPER_COMMIT}} from sfbrigade/data-covid19-sfbayarea
           https://github.com/sfbrigade/data-covid19-sfbayarea/commit/${{env.SCRAPER_COMMIT}}
         reviewers: kengoy
+
+    - name: Fail Workflow if the Scraper Had Errors
+      if: ${{ steps.scrape_data.outcome == 'failure' }}
```

Comment:
I wasn't sure which one below is the right if statement on the GitHub Action way... Let's merge it and run to see if works.

 ` if: ${{ steps.scrape_data.outcome }} == 'failure' `
or
 ` if: steps.scrape_data.outcome == 'failure' `
or
 `if: ${{ steps.scrape_data.outcome == 'failure' }}`